tests/onnxruntime/test_onnxruntime.py

```diff
@@ -53,7 +53,7 @@ def test_optimize(self):
             "roberta-base",
             "google/electra-small-discriminator",
         }
-        optimization_config = OptimizationConfig(optimization_level=99, optimize_with_onnxruntime_only=False)
+        optimization_config = OptimizationConfig(optimization_level=2, optimize_with_onnxruntime_only=False)
```

Comment:
Why 2 and not 1 here for `optimization_level`?

Reply:
For the tests, I think it's better to verify that even the extended graph optimizations don't impact too much the models outputs. The optimization level of [99](https://onnxruntime.ai/docs/performance/graph-optimizations.html#layout-optimizations) doesn't seem useful in our use cases.